app/models/services/no-op-service.ts

```diff
@@ -57,6 +57,7 @@ export default class NoOpService extends BaseService<void> {
       createdAt: now,
       updatedAt: now,
       message: this.message,
+      collectionIds: this.operation.collectionIds,
```

Comment:
I'm not sure if we want to display the collection IDs as part of the job status response.

Counterpoint - we have been adding fields that aren't part of the OGC processing spec and seeing the collections used for a job does seem like useful information, so maybe we keep this.

Reply:
Oh interesting. I thought by deleting it here https://github.com/nasa/harmony/blob/harmony-880/app/models/job.ts#L555 it would be inaccessible from job status endpoints.

Reply:
You're correct. I didn't fully expand this code snippet - I thought this was the output from the noOpService which mimics a job status response, but this is just constructing a job.